importer/src/utils.rs

```diff
@@ -111,10 +112,53 @@ pub fn osmconvert(
     );
 }
 
+/// Creates a RawMap from OSM and other input data.
+pub async fn osm_to_raw(
+    name: MapName,
+    timer: &mut abstutil::Timer<'_>,
+    config: &ImporterConfiguration,
+) -> RawMap {
+    let boundary_polygon = format!(
+        "importer/config/{}/{}/{}.poly",
+        name.city.country, name.city.city, name.map
+    );
+    let osm_url = crate::pick_geofabrik(boundary_polygon.clone())
```

Comment:
Before, we manually specified geofabrik URLs, which were found using this tool. Let's skip a step and just always run that tool to get the source URL.